docs/false_positives.md

```diff
@@ -0,0 +1,243 @@
+  int Val = 0;
+  ...
+  if (mystrlen(Str)) {
+    Val = 1;
+  }
+  return 5 / Val; // Division by zero.
+}
+```
+
+It can be rewritten to as the following to suppress the warning:
+
+```cpp
+int f() {
+  const char *Str = "MyStr";
+  int Val = 0;
+  ...
+  if (strlen(Str)) {
+    Val = 1;
+  }
+  return 5 / Val; // No warning.
+}
+```
+
+The result is easier to read for other developers who might not be familiar
+with the custom version of the function. The standard library functions also
+tend to be faster and more correct than custom solutions.
+
+#### <a name="do-not-turn-off-code-checks"></a> Do not turn off code checks
+
+Checks in the static analyzer are usually not just reporting issues but
+also help modelling language constructs and library functions. You should never
+turn off checks from the core package.
+
+#### <a name="suppress-specific-dead-store-warnings"></a> Suppress specific dead store warnings
+
+How to suppress a specific dead store warning and more useful tips can be found
+[here](https://clang-analyzer.llvm.org/faq.html).
+
+#### <a name="alternative-implementations"></a> Alternative implementations
+
+If there are huge source of false positives due to the analyzer can not model a
+function properly you could either disable the analyzer to analyze that
+function or provide an alternative implementation that is easier to model
+(i.e.: implementation without bitwise operation tricks). This can be achieved
+using the `__clang_analyzer__` macro.
+
+### <a name="syntax-based-checks"></a> Syntax based checks
+
+Clang tidy has lots of useful syntax based checks. Some of these checks
+find bugprone code snippets. When these snippets are intentional, usually
+there is a natural way to make the intent more explicit. Unfortunately,
+it is hard to give a general guideline, because the details are different for
+each check. The documentation of the check might contain hints how to
+express intention more clearly. Let us look at an example:
+
+  
+```cpp
+double f(int i) {
+  return 32 / (2 + i); // Warning, integer division, loss of precision.
+}
+```
+
+It can be rewritten to as the following to suppress the warning:
+
+```cpp
+double f(int i) {
+  return (int)(32 / (2 + i)); // No warning, the intention is explicit.
+}
+```
+
+The second version makes it clear even though the return value is a floating
+point value the loss of precision during integer division is intentional.
+Adding a comment why this is intentional would make this even clearer.
+Such edits makes the code easier to understand for fellow developers. 
+
+### <a name="suppress-results"></a> Suppress results
+
+When none of the above worked, we can suppress the finding. There are multiple
+ways to do that and it is important to chose the right one.
+
+#### <a name="third-party-code"></a> 3rd party code
+
+We usually have no control over 3rd party code and not interested in the
+findings in such code. CodeChecker supports skip files to skip certain parts
```

Comment:
"skip file" is the name of the argument and the descriptor used, but that is not the name of the feature. It makes this sentence clunky.

I think this works better: "CodeChecker supports skipping certain files (or even directory trees) of the analyzed project.".